src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,56 @@
+package duke;
+
+import duke.commands.AddCommand;
+import duke.commands.Command;
+import duke.commands.DeleteCommand;
+import duke.commands.DoneCommand;
+import duke.commands.ExitCommand;
+import duke.commands.FindCommand;
+import duke.commands.ListCommand;
+import duke.commands.WrongCommand;
+
+/**
+ * Deals with making sense of user's commands.
+ */
+public class Parser {
+
+    /**
+     * Returns the Command to be executed after reading
+     * in user's input.
+     * @param fullCommand User's input.
+     * @return Command.
+     */
+    public static Command parse(String fullCommand) {
+        String nextInput = fullCommand.toLowerCase();
+        if (nextInput.contains("delete")) {
```

Comment:
I think that Switch statements would make the code more readable as compared to multiple if else statements. What do you think?